scripts/issue/247/upgrade_arb_crv_gauges.py

```diff
@@ -0,0 +1,81 @@
+from decimal import Decimal
+
+from brownie import accounts, web3, interface, chain
+from rich.console import Console
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import ADDRESSES_ETH
+from helpers.addresses import registry
+
+console = Console()
+
+DEV_MULTI = registry.eth.badger_wallets.dev_multisig
+
+PBTC_STRATEGY_DEP = ADDRESSES_ETH["strategies"]["_deprecated"]["native.pbtcCrv"]["v1"]
+PBTC_STRATEGY = ADDRESSES_ETH["strategies"]["native.pbtcCrv"]
+
+OBTC_STRATEGY_DEP = ADDRESSES_ETH["strategies"]["_deprecated"]["native.obtcCrv"]["v1"]
+OBTC_STRATEGY = ADDRESSES_ETH["strategies"]["native.obtcCrv"]
+
+PNT = ADDRESSES_ETH["treasury_tokens"]["PNT"]
+BOR = ADDRESSES_ETH["treasury_tokens"]["BOR"]
+BORING = ADDRESSES_ETH["treasury_tokens"]["BORING"]
+
+STRATEGIES = [PBTC_STRATEGY_DEP, PBTC_STRATEGY, OBTC_STRATEGY_DEP, OBTC_STRATEGY]
+STRATEGY_MAPPING = {
+    PBTC_STRATEGY_DEP: "pbtcCrv_deprecated",
+    PBTC_STRATEGY: "pbtcCrv",
+    OBTC_STRATEGY_DEP: "obtcCrv_deprecated",
+    OBTC_STRATEGY: "obtcCrv",
+}
+
+TOKENS = [PNT, BOR, BORING]
+TOKEN_MAPPING = {
+    PNT: "PNT",
+    BOR: "BOR",
+    BORING: "BORING",
+}
+
+
+def main(simulation="false"):
+
+    safe = GreatApeSafe(DEV_MULTI)
+
+    for strategy in STRATEGIES:
+        console.print(f"[green]Sweeping {STRATEGY_MAPPING[strategy]}[/green]")
+        sweep_strategy(strategy, safe, simulation)
+
+    safe.post_safe_tx(call_trace=True)
+
+
+def sweep_strategy(
+    strategy_address: str,
+    safe: GreatApeSafe,
+    simulation: bool = True,
+):
+    strategy = interface.IStrategy(strategy_address, owner=safe.account)
+
+    controller_address = strategy.controller()
+    controller = interface.IController(controller_address, owner=safe.account)
+
+    for token_address in TOKENS:
+        token = interface.IERC20(token_address)
```

Comment:
```suggestion
        token = interface.ERC20(token_address)
```
`IERC20` is incompatible with `GreatApeSafe` since it doesnt have all attributes (ie `.symbol`)